admin/load.php

```diff
@@ -5,10 +5,38 @@
  * @package performance-lab
  */
 
+use PerformanceLab\Plugin_Manager;
+
 if ( ! defined( 'ABSPATH' ) ) {
 	exit; // Exit if accessed directly.
 }
 
+add_action( 'admin_enqueue_scripts', 'perflab_admin_scripts' );
+
+/**
+ * Callback function to handle admin scripts.
+ *
+ * @since n.e.x.t
+ *
+ * @return void
+ */
+function perflab_admin_scripts() {
+	wp_enqueue_script( 'updates' );
+
+	wp_localize_script(
+		'updates',
+		'_wpUpdatesItemCounts',
+		array(
+			'settings' => array(
+				'totals' => wp_get_update_data(),
+			),
+		)
+	);
+
+	wp_enqueue_script( 'thickbox' );
+	wp_enqueue_style( 'thickbox' );
+}
```

Comment:
This enqueues these scripts throughout the entire WP Admin, which is a bit wasteful and potentially bad for performance. We need to rely on the `$hook_suffix` parameter of the `admin_enqueue_scripts` action to only run this logic when it's the "hook suffix" for the correct screen (our plugin's screen).

I would recommend the following:
* Add the `admin_enqueue_scripts` action within `perflab_load_modules_page()` (which is called before that hook).
* Since that function will only be called for our admin page, you can add the callback unconditionally there, and it will still only enqueue the assets on our page.
* Rename this function to `perflab_enqueue_modules_page_scripts()` to clarify the intent, as it should only be used there.